schemas/remote_configuration_v1.schema.json

```
@@ -0,0 +1,40 @@
+{
+  "$id": "remote_configuration_v1.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "An object containing the set of rules used by the conditional gatherer.",
+  "descriptions": "A data structure used by the Insights Operator Gathering Conditions Service v1 API to return the remote configurations for the conditional gathering.",
```

I suppose you based on schema on my [cluster_version_mapping.schema.json](https://github.com/RedHatInsights/insights-operator-gathering-conditions/blob/main/schemas/cluster_version_mapping.schema.json)? :grinning: 

`descriptions` is an extra attribute that doesn't have an official meaning in the schema. The [cluster_version_mapping.schema.json](https://github.com/RedHatInsights/insights-operator-gathering-conditions/blob/main/schemas/cluster_version_mapping.schema.json) contains both `description` and `descriptions` by mistake. I'll fix it.

Here, I would suggest

```suggestion
  "description": "A data structure used by the Insights Operator Gathering Conditions Service v1 API to return the remote configurations for the conditional gathering.",
```